doctor/crud.go

```
@@ -90,21 +89,20 @@ func UpdateDoctor(w http.ResponseWriter, r *http.Request) {
 	filter := bson.M{"name": name, "contact": contact}
 	res := collection.FindOne(r.Context(), filter)
 	if res.Err() != nil {
-		log.Fatal("Error occured during finding doctor's name from DB", "error =", err)
+		log.Fatal("Error occured during finding doctor's name from DB", "error =", err, 404)
 	}
 	var docDB models.Doctor
 	err = res.Decode(&docDB)
 	if err != nil {
 		log.Fatal("Error occured during decoding doctor's name of DB", "error =", err)
 	}
-	
 
 	// 	res = collection.FindOneAndUpdate(r.Context(), filter, docReq)
 	// 	if res.Err() != nil {
 	// log.Fatal("Error occured during updating doctor's name", "error =", err)
 	// 	}
-
 	docDB.Name = docReq.Name
 	docDB.Contact = docReq.Contact
+	collection.UpdateOne(context.TODO(), filter, doc)
```

Don't ignore error here